scripts/Util.py

```diff
@@ -3061,10 +3061,10 @@ def __init__(self, current):
         self.url = None
         self.driver = None
         try:
-            cmd = "node -e \"require('./bin/HttpServer')()\""
+            httpServerCmd = "node node_modules/http-server/bin/http-server -p 8080 dist"
```

Comment:
We need a separate https server on port 9090. For WSS testing, I will look into it in a separate PR.